### DISCUSSION TITLE
How to make/pack a release version?

### DISCUSSION THREAD

**Discussion by gismofx** [adamralph/minver#862] [Q&A] [answered]
I seem to be missing something here...

I have made some commits, I added a tag 1.0.0. I want to build the release version of the NuGet. 

When I run `dotnet pack` from command line or UI, I get `blahblah.1.0.1-alpha.0.1.nupkg` Subsequent commits and packs do automatically bump the pre-release like `-alpha.0.x`  This part is working.

How can build 1.0.0. or 1.0.1 **release** package?

Thank you!

**Comment by bording** [comment, marked as answer]
If you tag a commit and then build that commit, you will get the RTM version number matching the tag. Make sure you are building the exact commit that you have tagged.

**Reply by gismofx** [reply]
Thanks! Got it!
